tools/cpp/unix_cc_configure.bzl

```diff
@@ -488,7 +488,8 @@ def configure_unix_toolchain(repository_ctx, cpu_value, overriden_tools):
             cc,
             "-xc++",
             cxx_opts + no_canonical_prefixes_opt + ["-stdlib=libc++"],
-        ),
+        ) +
+        ["/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk"],
```

Comment:
This is required because now this toolchain can be used when the user has Xcode + the CLT installed, in which case the Xcode paths are automatically collected, but this SDK can still be used.

Reply:
Can you add that as a comment in the code?